tests/test_stack.py

```diff
@@ -808,6 +859,11 @@ def test_get_role_arn_without_role(self):
     def test_get_role_arn_with_role(self):
         assert self.stack._get_role_arn() == {"RoleARN": sentinel.role_arn}
 
+    def test_get_stack_timeout(self):
+        assert self.stack._get_stack_timeout() == {
+            "TimeoutInMinutes": sentinel.stack_timeout
+        }
+
```

Comment:
Although I am a fan of tests in general, I'm not entirely convinced this test is necessary.

Reply:
I'd also like to get an opinion from @theseanything, especially the tests.

Reply:
It does not hurt though 😄

Reply:
@afraisse, perhaps, but since that function being tested is obviously correct, I feel the ratio of utility of the test to the upkeep cost is not in its favor (any, even trivial change to that function, will require an update to that test, even if it's still obviously correct).

Reply:
No argument here. I added it because of 
```python
def test_get_role_arn_with_role(self):
    self.stack._get_role_arn() == {"RoleARN": sentinel.role_arn}
```
I'll gladly remove it.